unstoppable-ios-app/domains-manager-ios/Modules/Messaging/Chat/Cells/ChatTextCell/ChatTextCell.swift

```diff
@@ -14,7 +14,7 @@ final class ChatTextCell: UICollectionViewCell {
     @IBOutlet private weak var timeLabel: UILabel!
     
     private var textViewSideConstraints: [NSLayoutConstraint] = []
-    private var sender: ChatSender?
+    private var sender: MessagingChatSender?
```

Comment:
Isn't `MessagingChat` redundant? Simply `ChatSender` was better, if there are not specific reason for this

Reply:
I've decided to make naming convention for all messaging-related entities to start with Messaging. MessagingChat, MessagingChatMessage, etc. It felt confusing with old naming convention in my experience .